### PR TITLE
databases-week2/Sevda

### PR DESCRIPTION
**Pull request by Sevda-NZ**
Thanks for the review :) I am still working on part-4

### CODE REVIEW

File: 5_databases/week2/part1.sql [hunk 1]
```diff
@@ -0,0 +1,26 @@
+--Add a task with these attributes: title, description, created, updated, due_date, status_id, user_id
+INSERT INTO task (title, description, created, updated, due_date, status_id) VALUES ("Become a developer", "Work hard", "2017-10-06 04:03:52", "2017-09-20 19:34:43", "2017-12-22 20:58:03", 2)
+
+--Change the title of a task
+UPDATE task
+SET title = "go to hyf"
+WHERE title = "Buy gift for Paul";
```
Comment (adionditsak) [inline 5_databases/week2/part1.sql:7]: Correct, but better to do with a primary key (id) so you dont edit more than one by accident

File: 5_databases/week2/part1.sql [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+--Change a task due date
+UPDATE task
```
Comment (adionditsak) [inline 5_databases/week2/part1.sql:10]: Correct, but better to do with a primary key (id) so you dont edit more than one by accident

File: 5_databases/week2/part1.sql [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+SET due_date = "2020-12-01 13:28:35"
+WHERE due_date = "2017-12-01 13:28:35";
+
+--Change a task status
+UPDATE task
```
Comment (adionditsak) [inline 5_databases/week2/part1.sql:15]: for this and next, just keep it simple and remove `task.` before column names eg. status_id

File: 5_databases/week2/part2.sql [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+--Add a new column to the class table named status which can only have the following values: not-started, ongoing, finished (hint: enumerations)
+
+ALTER TABLE Class
+ADD
+    COLUMN status ENUM(
+        'Not started',
+        'In progress',
+        'Done'
+    ) DEFAULT 'Not started';
```
Comment (adionditsak) [inline 5_databases/week2/part2.sql:42]: Well done

File: 5_databases/week2/part3.sql [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+WHERE user.name = 'Donald Duck' AND status.name = 'Not started';
+
+
+--Get all the tasks for 'Maryrose Meadows' that were created in september (hint: month(created)=month_number)
+
+SELECT task.*
+FROM user
+JOIN user_task ON user.id = user_task.user_id
+JOIN task ON user_task.task_id = task.id
+WHERE user.name = 'Maryrose Meadows' AND MONTH(task.created) = 9;
```
Comment (adionditsak) [inline 5_databases/week2/part3.sql:27]: Good job

File: 5_databases/week2/part3.sql [hunk 6]
```diff
@@ -0,0 +1,34 @@
+-- Active: 1699187035090@@127.0.0.1@3306@HYF_lesson2
+USE hyf_lesson2;
+
+--Get all the tasks assigned to users whose email ends in @spotify.com
+
+SELECT * 
+FROM user
```
Comment (adionditsak) [inline 5_databases/week2/part3.sql:7]: Here you should use the user_task many-to-many table to make relations like you do below so you get the task and not the user

**Comment by adionditsak** [comment]
@Sevda-NZ How is it going, let me know if you need help?

**Comment by Sevda-NZ** [comment]
implemented the reviews :)

**Pull request closed without merging**